server/odc-service/src/main/java/com/oceanbase/odc/service/task/executor/server/TaskMonitor.java

```diff
@@ -216,13 +218,14 @@ private void uploadLogFileToCloudStorage(DefaultTaskResult finalResult) {
         if (cloudObjectStorageService != null && cloudObjectStorageService.supported()
                 && JobUtils.isK8sRunModeOfEnv()) {
             LogBiz biz = new LogBizImpl();
-            Map<String, String> logMap = null;
+            Map<String, String> logMap = new HashMap<>();
             try {
                 logMap = biz.uploadLogFileToCloudStorage(finalResult.getJobIdentity(), cloudObjectStorageService);
+                logMap.put(JobAttributeKeyConstants.UPLOAD_LOG_STATUS, UploadLogStatus.SUCCESS);
             } catch (Throwable e) {
                 log.warn("Upload job log file to cloud storage occur error, jobId={}", getJobId(), e);
                 // putAll will throw NPE if it returns null.
-                logMap = new HashMap<>();
+                logMap.put(JobAttributeKeyConstants.UPLOAD_LOG_STATUS, UploadLogStatus.FAILED);
```

Comment:
may use exists LOG_STORAGE_FAILED_REASON attribute

Reply:
removed and fixed the issue of old fields were not effective